packages/adapters/wagmi/src/tests/client.test.ts

```diff
@@ -101,11 +104,14 @@ describe('WagmiAdapter', () => {
       expect(adapter.namespace).toBe('eip155')
     })
 
-    it('should set wagmi connectors', () => {
+    it('should set wagmi connectors', async () => {
       vi.spyOn(wagmiCore, 'watchConnectors').mockImplementation(vi.fn())
 
       adapter.syncConnectors({ networks: [mainnet], projectId: 'YOUR_PROJECT_ID' }, mockAppKit)
 
+      // Wait for connectors to be set
+      await new Promise(resolve => setTimeout(resolve, 200))
```

Comment:
this is a bit weird, shouldn't we make `syncConnectors` async instead and await it? 
Too much hustle?